src/dialogs/more-info/controls/more-info-input_timetable.ts

```diff
@@ -0,0 +1,188 @@
+import "@material/mwc-icon-button/mwc-icon-button";
+import "../../../components/ha-svg-icon";
+import "../../../components/ha-switch";
+import type { HaSwitch } from "../../../components/ha-switch";
+import "../../../components/paper-time-input";
+import { mdiDelete, mdiPlus } from "@mdi/js";
+import { HassEntity } from "home-assistant-js-websocket";
+import { BINARY_STATE_ON, BINARY_STATE_OFF } from "../../../common/const";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import { HomeAssistant } from "../../../types";
+
+declare type Event = {
+  time: string;
+  state: string;
+};
+
+@customElement("more-info-input_timetable")
+class MoreInfoInputTimetable extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({
+    attribute: false,
+    hasChanged(newVal: HassEntity, oldVal: HassEntity) {
+      if (!newVal) {
+        return false;
+      }
+      return (
+        !oldVal ||
+        JSON.stringify(oldVal.attributes.timetable) !==
+          JSON.stringify(newVal.attributes.timetable)
+      );
+    },
+  })
+  public stateObj?: HassEntity;
+
+  private _timetable: Array<Event> = [];
+
+  private _disable_reconfig = false;
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this.stateObj) {
+      return html``;
+    }
+    this._disable_reconfig = true;
+    return html`
+      ${this._timetable.map((event: Event, index: number) =>
+        this._renderEntry(event, index)
+      )}
+      <div class="add">
+        <mwc-icon-button
+          label="Add"
```

Comment:
```suggestion
          label="${this.hass!.localize("ui.dialogs.more_info_control.input_timetable.add")}"
```
They should the localized as well, so reuse your existing translation keys

Reply:
Done